arroyo/processing/strategies/run_task_with_multiprocessing.py

```diff
@@ -608,6 +611,11 @@ def __submit_batch(self, input_block_too_small: bool) -> None:
         )
         self.__batch_builder = None
 
+        if self.__last_batch_time == 0.0:
+            self.__last_batch_time = time.time()
+        else:
+            self.__metrics.timing("arroyo.strategies.run_task_with_multiprocessing.batch.build_and_submit_time", time.time() - self.__last_batch_time)
```

Comment:
I think you also need to reset self.__last_batch_time here.

Reply:
Oops forgot that line. I think we need to reset it to the current time (so that we have the time that the previous batch was submitted)